src/Univ2LpOracle.t.sol

```diff
@@ -645,4 +646,54 @@ contract UNIV2LPOracleTest is DSTest {
 
         assertTrue(thirdVal > secondVal);                               // Verify price of WBTC0ETH LP token increased after trade
     }
+
+    function testCurSlot0x3() public {
+        daiEthLPOracle.poke();                                       // Poke oracle
+        hevm.warp(add(daiEthLPOracle.zzz(), daiEthLPOracle.hop()));  // Time travel into the future
+        daiEthLPOracle.poke();                                       // Poke oracle again
+        daiEthLPOracle.kiss(address(this));                          // Whitelist caller
+        (bytes32 val, bool has) = daiEthLPOracle.peek();             // Peek oracle price without caller being whitelisted
+        assertTrue(has);                                             // Verify oracle has value
+        assertTrue(val != bytes32(0));                               // Verify peep returned valid value
+
+        // Load memory slot 0x3
+        // Keeps `cur` slot parity with OSMs
+        bytes32 curPacked = hevm.load(address(daiEthLPOracle), bytes32(uint256(3)));
+
+        bytes16 memhas;
+        bytes16 memcur;
+        assembly {
+            memhas := curPacked
+            memcur := shl(128, curPacked)
+        }
+
+        assertTrue(uint256(uint128(memcur)) > 0);          // Assert nxt has value
```

Comment:
This comment is maybe bigger than this PR, but it seems unnecessarily imprecise and fragile to test things like the location of storage slots while relying on reading data from on-chain.